docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1491,7 +1388,6 @@ The global configuration contains non-tenant specific configuration parameters (
             },
             "jdbc":{
                 "__comment":"Configuration properties for the JDBC persistence implementation",
-                "bootstrapDb":false
```

Comment:
Does this make sense to have this section. now? or maybe update the comment to indicate it is a placeholder?

Reply:
I don't think it hurts to have it as a placeholder; and I'd prefer not to touch the user's guide any more in this change set because I have extensive changes in https://github.com/IBM/FHIR/pull/1989